src/components/reusables/steps.ts

```diff
@@ -7,52 +7,70 @@ export const tourSteps: Tour[] = [
             {
                 icon: "👋",
                 title: "Welcome to AIBTC.DEV.",
-                content: "I'm your guide to help you get started..",
+                content: "I'll guide your through the basics to get started. Let's dive in!",
                 selector: "#step1", //now you can assign any component this id and it will guide you to this step...
                 side: "bottom", //position where you want to show the guide card 
                 showControls: true, //default 
                 showSkip: true, //default
             },
             {
-                icon: "💬",
-                title: "Threads",
-                content: "It consist your threads which is basically a chat history",
+                icon: "",
+                title: "Chats 💬",
+                content: "All your past conversations lives here.",
                 selector: "#step2",
                 side: "right",
                 showControls: true,
                 showSkip: true,
             },
             {
-                icon: "💬",
-                title: "Add thread",
-                content: "You can add new thread by clicking this button and start a chat.",
+                icon: "",
+                title: "Start a new chat 💬",
+                content: "You can start a new chat from here.",
                 selector: "#step3",
                 side: "right",
                 showControls: true,
                 showSkip: true,
             },
             {
-                icon: "🧭",
-                title: "Navigations",
-                content: "You can create agents, DAOs and go to your profile from here..",
-                selector: "#step4",
+                icon: "",
+                title: "Agents 🤖",
+                content: "View, manage and assign tasks to your AI agents.",
+                selector: "#agents",
                 side: "right",
                 showControls: true,
                 showSkip: true,
             },
             {
-                icon: "👜",
-                title: "Wallet",
-                content: "These is your personal wallet.",
+                icon: "",
+                title: "DAOs 🏛️",
```

Comment:
~~Why is the icon/title diff here from the others?~~

Why do we use a blank icon for some and put it in the title instead?